### PR TITLE
New minimap

### PR DESCRIPTION
**Pull request by chasulin**
`County Map` in storybook

### CODE REVIEW

**Review by pnavarrc — COMMENTED** [review]
Commenting for now until we can sync!

File: src/components/NewLocationPage/CountyMap/CountyMap.style.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const desktopMapWidth = 320;
+
+export const PinnedContainer = css`
+  position: fixed;
+  top: 80px;
+  right: 16px;
+  box-shadow: 0px 2px 16px rgba(0, 0, 0, 0.08);
+  z-index: 901;
+`;
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/NewLocationPage/CountyMap/CountyMap.style.tsx:11]: the pinned to the top corner version for screens 1350px+ 

Screen Shot 2021-04-22 at 4 16 36 PM

File: src/components/NewLocationPage/CountyMap/CountyMap.style.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  }
+`;
+
+// The default aspect-ratio for the state and US maps is 800x600, so we
+// can use the width to calculate the height of the map container, which
+// makes it possible to center it vertically.
+export const MapContainerInner = styled.div<{ mapWidth: number }>`
+  width: ${({ mapWidth }) => `${mapWidth}px`};
+  height: ${({ mapWidth }) => `${0.75 * mapWidth}px`};
+`;
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/NewLocationPage/CountyMap/CountyMap.style.tsx:31]: i feel like the current implementation is simpler, no?

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**